.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+- [ ] Closes #xxxx
+- [ ] Tests added / passed
+- [ ] Passes `flake8`
```

Comment:
I've opted to:
- follow the checks as per https://github.com/dask/dask-ml/blob/main/ci/code_checks.sh (e.g., and not do `flake8 dask_ml` instead of just `flake8`);
- split the individual code checks into several lines.

Would be happy to amend the PR accordingly, subject to expressed preferences for alternatives.